### PR TITLE
Basic landing page

### PR DESCRIPTION
**Pull request by onebarloop**
First PR for my Capstone Project. You are welcome to check the User Story for this PR here: https://github.com/onebarloop/capstone-project/issues/1

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+import Main from "../components/Main";
+import Footer from "../components/Footer";
+import Header from "../components/Header";
 
 export default function Home(): JSX.Element {
   return (
     <>
       <Head>
         <title>Wannado</title>
         <meta name="description" content="Generated by create next app" />
```
Comment (MerleMak) [inline pages/index.tsx]:
```suggestion
```

File: globalStyles.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyles = createGlobalStyle`
+   
+   @font-face {
+    font-family: Inconsolata;
+    src: url('/fonts/Inconsolata/Inconsolata-Regular.ttf') format("truetype");
```
Comment (MerleMak) [inline globalStyles.ts:7]: 😁🙌🏽

File: components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export default function Footer(): JSX.Element {
+  return <StyledFooter></StyledFooter>;
```
Comment (MerleMak) [inline components/Footer.tsx:4]: Usually we try to only commit code that is already useful for the user. Empty you could just leave uncommited until you fill them with content :)

File: components/Picture.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+type PictureProps = {
+  width: number;
+  length: number;
+  source: string;
+};
```
Comment (MerleMak) [inline components/Picture.tsx:8]: 😎

File: components/Picture.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import Image from "next/image";
+
```
Comment (MerleMak) [inline components/Picture.tsx:3]: Its best to add a comment for the reader here that you are working with typescript :))

**Review by MerleMak — COMMENTED** [review]
Really good job so far! 
I left some small remarks for you.
keep up the good work :)👍🏼

File: components/Footer.tsx (hunk 3 above)
Comment (onebarloop) [reply components/Footer.tsx:4]: Before I pushed, it said 'Footer' 😅

But yeah, I see the point!

File: components/Picture.tsx (hunk 5 above)
Comment (onebarloop) [reply components/Picture.tsx:3]: I will do that before the next push, thank you!

**Pull request merged**